File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/paths/userusername/delete/Parameters.java
package org.openapijsonschematools.client.paths.userusername.delete;

import org.openapijsonschematools.client.paths.userusername.delete.parameters.Parameter0;

import org.openapijsonschematools.client.parameter.Parameter;

import java.util.Map;
import java.util.AbstractMap;

public class Parameters {

    public static class PathParametersSerializer extends PathSerializer {
        public PathParametersSerializer() {
            super(
                Map.ofEntries(
                    new AbstractMap.SimpleEntry<>("", new Parameter0.Parameter01())
                )
            );
        }
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/paths/storeorderorderid/delete/Parameters.kt
package org.openapijsonschematools.client.paths.storeorderorderid.delete;

import org.openapijsonschematools.client.paths.storeorderorderid.delete.parameters.Parameter0;
import org.openapijsonschematools.client.parameter.PathSerializer;

import java.util.Map;
import java.util.AbstractMap;

public class Parameters {

    public static class PathParametersSerializer extends PathSerializer {
        public PathParametersSerializer() {
            super(
                Map.ofEntries(
                    new AbstractMap.SimpleEntry<>("order_id", new Parameter0.Parameter01())
                )
            );
        }
    }
}
